src/main/java/seedu/address/logic/commands/AddPetCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PET_AGE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PET_GENDER;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PET_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.pet.Pet;
+import seedu.address.model.pet.exceptions.DuplicatePetException;
+
+/**
+ * Adds support for adding a pet into the program.
```

Comment:
This should just describe what this command does. (e.g. `Adds a pet into the application.`)